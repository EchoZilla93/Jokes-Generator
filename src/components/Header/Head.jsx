import React from 'react';
import './headStyles.scss';

export const Head = () =>{
    return(
        <div className="header">
            <div className="header__logo">
                msi 2020
            </div>
            <div className="header__welcomesTxt">
                <h2 className="header__welcomesTxt-1stfloor">Hey!</h2>
                <h3 className="header__welcomesTxt-2stfloor">Let's try to find a joke for you:</h3>
            </div>
        </div>
    )
}

export default Head;