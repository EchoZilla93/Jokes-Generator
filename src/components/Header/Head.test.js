import React from 'react';
import {Head} from './Head';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';

describe('Head component', () => {
    it('renders expected snapshot', () => {
        const wrapper = shallow(<Head/>);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
})