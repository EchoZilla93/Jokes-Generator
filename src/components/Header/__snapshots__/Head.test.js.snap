// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Head component renders expected snapshot 1`] = `
<div
  className="header"
>
  <div
    className="header__logo"
  >
    msi 2020
  </div>
  <div
    className="header__welcomesTxt"
  >
    <h2
      className="header__welcomesTxt-1stfloor"
    >
      Hey!
    </h2>
    <h3
      className="header__welcomesTxt-2stfloor"
    >
      Let's try to find a joke for you:
    </h3>
  </div>
</div>
`;
